src/main/java/uk/gov/gchq/palisade/service/PolicyCacheWarmerFactory.java

```diff
@@ -43,9 +47,17 @@
     /**
      * Creates a {@link Policy} using the data within an implementation of the {@link PolicyCacheWarmerFactory}.
      *
-     * @return the {@link Policy} that has been created.
+     * @param users     a {@link List} of {@link UserCacheWarmerFactory} implementations
+     * @return          the {@link Policy} that has been created.
```

Comment:
update this return description now it has changed

Reply:
Updated